### ISSUE TITLE
Can i run Spark NLP translator offline on cpu and Windows 11?

### ISSUE THREAD

**Issue by JohnClaw** [JohnSnowLabs/spark-nlp#13865]
### Description

 I am searching for translator that can translate from Uralic languages to English and found in google that Spark NLP can do this.

### Preferred Solution

Does exist any Windows gui or web ui for your translator?

### Additional Context

Is there any Windows installer for Spark NLP?

**Comment by maziyarpanahi** [comment]
Yes, the library was designed to operate 100% offline. (in fact, we only use the Internet to download models, if you have them on the disk you just load them offline)

- Keep in mind, this is a library, not software to have a GUI or installer (there is a small learning curve like any other library). Windows installation: https://sparknlp.org/docs/en/install#windows-support
- Offline example: https://github.com/JohnSnowLabs/spark-nlp#offline
